cmake/std/atdm/README.md

```diff
@@ -1034,6 +1035,39 @@ $ bsub -x -Is -n 20 \
 ```
 
 
+### vortex
+
+Once logged on to 'vortex' (SRN), one can either build and configure on the
+login node or the compute node. Make sure to setup SSH keys as described in
+`/opt/VORTEX_INTRO` before trying to build on a compute node. To get a list of
+compute nodes, run `bhosts`, then simply ssh to a given hostname. For example to
+configur, build and run the tests for the default `cuda-debug` build for
+`kokkos` (after cloning Trilinos on the `develop` branch), do:
+
+```bash
+$ cd <some_build_dir>/
+
+$ source $TRILINOS_DIR/cmake/std/atdm/load-env.sh cuda-debug
+
+$ cmake -G'Unix Makefiles' \
+  -DTrilinos_CONFIGURE_OPTIONS_FILE:STRING=cmake/std/atdm/ATDMDevEnv.cmake \
+  -DTrilinos_ENABLE_TESTS=ON \
+  -DTrilinos_ENABLE_Kokkos=ON \
+  $TRILINOS_DIR
+
+$ make NP=20
+
+$ ctest
```

Comment:
This is not really true if you build on a login node, is it?  What is the exact command to run ctest on a compute node?

Reply:
Also, technically, should they not run `ctest -j4` because that is what the Jenkins build does, no?

Reply:
> This is not really true if you build on a login node, is it?

Is is.

Reply:
> Also, technically, should they not run `ctest -j4` because that is what the Jenkins build does, no?

I will check the Jenkins build and update accordingly.